### ISSUE TITLE
SVG:  ^.class := has no effect

### ISSUE THREAD

**Issue by kschuetz** [japgolly/scalajs-react#287]
Hello.  Great work on this library so far!

I've encountered a problem when working with SVG elements.  There seems to be a problem with setting the class attribute on them.

For example:
```
<.rect(
        ^.`class` := "foobar",
        ^.x := 0,
        ^.y := 0,
        ^.width := 100,
        ^.height := 100
      )
```
This would be expected to set the class attribute on the rect element to "foobar", but the element ends up having no class attribute.
So far, this seems to be the case for all SVG elements I've tried.

Thanks!

**Comment by kschuetz** [comment]
I've found a temporary workaround:

```
<.rect(
        ReactAttr.ClassName := "foobar",
        ^.x := 0,
        ^.y := 0,
        ^.width := 100,
        ^.height := 100
      )
```

**Comment by ramnivas** [comment]
Did you try:

```scala
<.rect(
        ^.className := "foobar",
        ^.x := 0,
        ^.y := 0,
        ^.width := 100,
        ^.height := 100
      )
```

I believe `cls` also works.

**Comment by kschuetz** [comment]
Yes.  `className` and `cls` work for HTML tags, but for SVG tags it doesn't compile.

**Comment by kschuetz** [comment]
I've found that my problem was that I included the following import:

`import japgolly.scalajs.react.vdom.svg.prefix_<^._`

This contains brings all of the SVG tags and attributes into scope, but it brings none of the HTML or React attributes.

The solution was to replace it with the following import:

`import japgolly.scalajs.react.vdom.prefix_<^._`

and to refer to all of the SVG tags and attributes explicitly using `<.svg` and `^.svg`.

The following gets the job done:

```
<.svg.circle(
          ^.`class` := "foobar",
          ^.key := 1,
          ^.svg.cx := cx,
          ^.svg.cy := cy,
          ^.svg.r := 0.06
        )
```

This also solves the problem in [issue 289](https://github.com/japgolly/scalajs-react/issues/289).
